server/server.go

```diff
@@ -112,3 +113,73 @@ func (s *Server) SumCalories(id string) (int, error) {
 	}
 	return sum, nil
 }
+
+// CheckDB : return a boolean check on the database structure
+func (s *Server) CheckDB() bool {
```

Comment:
This function might be better named `checkDatabaseIntegrity`, aka make it private, and instead of calling it in `main.go` call it it `Server.New()`, that way we can avoid exposing this one-time function

Reply:
Oh yea I was gonna ask you about that. I should just create a new var called svr or something and check return it or nil if wrong. Right?

Reply:
don't return nil - just `log.Fatal` it